.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 dist
+*.log
```

Comment:
So I stop trying to merge in my npm error *.log files whenever npm vomits all over the place.